third_party/p6/p6_sdk.gni

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+import("//build_overrides/jlink.gni")
+import("//build_overrides/mbedtls.gni")
+
+import("//build_overrides/p6.gni")
+import("p6_board.gni")
+
+# TODO: Don't hardcode config/toolchain
+# TODO: Convert to GN action?
+if (is_debug) {
+  debug_str = "Debug"
+} else {
+  debug_str = "Release"
+}
+mtb_json = exec_script("$p6_sdk_build_root/p6_sdk/make_gni.py",
```

Comment:
This runs "make" ?

We shouldn't be running another build system inside exec_script().

Reply:
@mspang, as stated in the reply re: using make for flashing, our SDK includes commands encoded as make targets. In the case of the make_gni.py script, that script uses the make system from our SDK to discover libraries to be cloned from our SDK on GitHub for integration into Matter. Once the libraries are cloned, the make_gni.py script then generates a set of JSON files that describe the files and include paths coming from our SDK. That description is then used in the Matter GN build flow.

We don't use make to do any of the product build, only to acquire and describe that which will be built by Matter GN.

Thank you,

Ian

Reply:
@mspang  : can we resolve this ?